autoload/neomake/makers/ft/dockerfile.vim

```diff
@@ -0,0 +1,10 @@
+function! neomake#makers#ft#dockerfile#EnabledMakers() abort
+    return executable('hadolint') ? ['hadolint'] : []
```

Comment:
I am using this wrapper myself:
```sh
#!/bin/sh

docker run --rm -i hadolint/hadolint hadolint "${@:--}"
```

I could imagine having better support for using Docker images as programs for makers in general.
E.g. so that `docker:hadolint/hadolint` could be specified / configured as executable directly.

(it should already be possible to use `docker` as `exe` and adjust the args however)

What do you think?

Reply:
Let's keep it in mind for the future.  Having a wrapper (or installing it yourself actually) is good enough for now.